src/js/controllers/mapController.ts

```diff
@@ -258,6 +259,8 @@ export class MapController {
           });
 
           this.initializeAndSetSketch();
+          this.initializeAndSetVIIRSLayers();
+          this.initializeAndSetMODISLayers();
```

Comment:
Just noting that we've discussed conditionally firing these functions if the user's config files have `VIIRS` or `MODIS`. We've also discussed maintaining a single source of truth RE: available layers, instead of adding them directly to `this._map`